app/account/forms.py

```diff
@@ -80,6 +80,16 @@ def validate_email(self, field):
             raise ValidationError('Unknown email address.')
 
 
+class CreatePasswordForm(Form):
+    password = PasswordField('Password', validators=[
```

Comment:
I think @aharelick mentioned we might want to impose a minimum password length. I think that's a good idea, and shouldn't be too difficult.